sample_project/sample.tf

```diff
@@ -1,34 +1,34 @@
-variable "aiven_api_token" {}
-variable "aiven_card_id" {}
-variable "aiven_project_name" {}
+variable "aiven_api_token" {
+  type = string
+}
+
+variable "aiven_project_name" {
+  type = string
+}
 
 terraform {
   required_providers {
     aiven = {
-      source = "aiven/aiven"
-      version = "2.X.X"
+      source  = "aiven/aiven"
+      version = "2.1.13"
```

Comment:
Let's keep `version = "2.X.X"` for consistency with other examples?

Reply:
ok!